frontend/components/campaigns/form/CampaignFormCharitySection.tsx

```diff
@@ -25,7 +27,7 @@ const CampaignFormCharitySection = ({ index, charityOptions, handleRemove }: Pro
           name={`${arrayFieldName}.charity.id`}
           label="Find Charity"
           placeholder="Enter the charity name"
-          options={charityOptions}
+          options={removeSelectedCharitiesFromOptions(charityOptions, selected, index)}
```

Comment:
Is it possible to do the filtering in the parent component? So in `CampaignFormCharitiesSection`, we remove all selected charities, then we pass the filtered options to this component

And then in this component we add the selected charity for this section to the top of the filtered options

Reply:
I understand the point of moving to parent, but I'd rather not prepend on an array that needs to be re-rendered very frequently. So I just moved the function to the parent comment, and left it as it is (including needing index)